superbench/runner/playbooks/check_env.yaml

```diff
@@ -26,7 +26,7 @@
     sb_nodes: '{{ hostvars.values() | map(attribute="ansible_hostname") | sort }}'
     sb_env: |
       # sb env
-      SB_WORKSPACE='{{ workspace if skip_docker else "/root" }}'
+      SB_WORKSPACE={{ workspace if skip_docker else '/root' }}
```

Comment:
Why related to IB?

Reply:
because ib scan benchmark will read host file in this path to generate the pairs, other benchmarks do not need that file